pkgs/by-name/so/soundconverter/package.nix

```
@@ -0,0 +1,117 @@
+{
+  lib,
+  fetchurl,
+  # Optional due to unfree license.
+  faacSupport ? false,
+  glib,
+  python3Packages,
+  gtk3,
+  wrapGAppsHook,
+  python3,
+  gsettings-desktop-schemas,
+  intltool,
+  xvfb-run,
+  gobject-introspection,
+  gst_all_1,
+  fdk-aac-encoder,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "soundconverter";
+  version = "4.0.5";
+
+  src = fetchurl {
+    url = "https://launchpad.net/soundconverter/trunk/${version}/+download/${pname}-${version}.tar.gz";
+    sha256 = "1wvrxcvbhgqx9kliz5v2kvc4w08aw043v8ph4kgfbp2xd0v5mc3r";
```

```suggestion
    hash = "...";
```

replacing ... with the SRI hash